src/azure-cli/azure/cli/command_modules/mysql/custom.py

```diff
@@ -1032,6 +1032,7 @@ def flexible_server_update_custom_func(cmd, client, instance,
 
     if high_availability:
         if high_availability.lower() != "disabled":
+            user_confirmation("Enabling High-availability may result in a short downtime for the server based on your server configuration. Do you want to continue and enable High availability?", yes=yes)
```

Comment:
This may break existing scripts for automation scenarios. Can you double confirm and provide some business justification since this is not regular breaking change window